lib/request-handler.js

```diff
@@ -44,6 +44,10 @@ const globalQueryParams = new Set([
   'link',
   'colorA',
   'colorB',
+  'prefix',
+  'suffix',
+  'uri',
+  'query'
```

Comment:
You can actually override these, for just this badge, in `server.js`. There are a couple others in there that do.

Reply:
Ah thanks! didn't realise, will update it now.